### DISCUSSION TITLE
[Question] Are there plans to implement stylus support for Android?

### DISCUSSION THREAD

**Discussion by RexLatro** [exult/exult#357] [General]
Sorry, I'm not sure if this is the right place to ask (I saw on the Exult page that the forum has been shut down and questions happen here), but I'm running Exult on my Z Fold 3 with stylus pen.  I'm able to use the stylus for click-and-drag or left-click, but there seems to be no ability or option to allow for the button built into my stylus to act as a right click.  

I've seen this ability supported in builds of SCUMM and other programs, but I'm not sure if it's a complicated feature to request.  Are there currently any other options to recreate a right-click input in the Android version of Exult?  I've tried gestures such as two-fingers/long-click/quick tap to no success, and have not found any information on either old Exult forums or Reddit

**Comment by DominusExult** [comment]
yeah, so far it has not been supported, thought of.

I don't know if SDL2 supports that as we rely on that to pass things through. It would make sense however that if you press the button and move the stylus, it moves the party (other than moving around (and right double click for moving to a spot) we don't use the right mouse button (I think).

**Reply by RexLatro** [reply]
Makes sense, and thanks for the reply!  Right now I've been able to use a bluetooth controller for movement and my finger to click and move objects, but it feels a bit awkward compared to the idea of having a right-click input on stylus.  Glad to know it's something that's been thought about
